features/graph/documentation/README.md

```diff
@@ -1,38 +1,82 @@
+
+## Requirements
+
+Netlify Graph uses the Netlify CLI to communicate between your local project and your Netlify site. Before you do anything else, you should install the latest version of the Netlify CLI.
+
+``` bash
+npm install netlify-cli -g
+```
 
-We handle the messy integration work, and you can focus on solving interesting problems. Instead of connecting different APIs and SDKs with brittle code, you can now abstract all that out behind a convenient GraphQL interface.
+## Example repo and tutorial
 
-## Components
+<!-- TODO: Update this section with link to the gravity repo -->
+If you'd like to try Netlify Graph in a test project, check out the `gravity` sample repo. It has a full tutorial that walks you through getting the `gravity` site running on Netlify to creating and using GraphQL queries.
 
-Netlify Graph in its current state has three core components, that you can learn more about by reading related documentation pages:
+## Get started with Netlify Graph
 
-- [API Management](api-management.md)
-- [API Explorer](api-explorer.md)
-- [API Authentication](api-authentication.md)
+  1. [Link your local project directory to your Netlify site](authentication.md#link-your-local-project-directory-to-your-netlify-site)
+  2. [Connect to your first API or service](authentication.md#connect-to-your-first-api-or-service) in the Netlify UI.
+  3. [Start a local CLI session with Netlify Dev](authentication.md#start-a-local-cli-session-with-netlify-dev).
+  4. [Create a GraphQL query with Graph Explorer](authentication.md#create-a-graphql-query-with-graph-explorer).
+  5. [Generate a query handler](authentication.md#generate-a-query-handler).
+  6. [Use a query handler in your project](authentication.md#use-a-query-handler-in-your-project).
 
-## Enabling the feature
+## Queries, mutations, and subscriptions
+<!-- QUESTION: Are we going to address fragments? -->
+<!-- QUESTION: Is my assumption that these are standard GraphQL correct? -->
+You use standard GraphQL queries, mutations, and subscriptions to interact with your Netlify Graph data.
 
-To enable Netlify Graph in your account, you need to use Netlify Labs, where you can click on **Enable** next to **Netlify Graph**.
+- **[Queries](https://graphql.org/learn/queries/).** Are how you get data.
+- **[Mutations](https://graphql.org/learn/queries/#mutations).** Are how you modify data. 
+- **[Subscriptions](https://www.onegraph.com/docs/subscriptions.html).** Let you receive data in response to an event.
 
-![Enabling Netlify Graph in Netlify Labs](../../../media/graph/enable-labs.gif)
+<!-- TODO: Possibly add examples, though I think linking to external sources is probably the best way to go. If we want to expand this section, we could have subsections for each type. If we want to spend the time on it and see value in it, we could also have a separate page dedicated to more in-depth explanation of these. -->
```

Comment:
I think we could expand on these and their differences in the future if we think there's value there. For now, I believe linking out to third-party info is better.